stable_baselines3/common/vec_env/subproc_vec_env.py

```diff
@@ -155,13 +160,12 @@ def close(self) -> None:
             process.join()
         self.closed = True
 
-    def get_images(self) -> Sequence[np.ndarray]:
+    def get_render_output(self) -> Sequence[Optional[np.ndarray]]:
```

Comment:
same, we should keep `get_images` but raise an error if `render_mode` is not `"rgb_array"`